x-pack/plugins/ml/public/application/settings/filter_lists/components/delete_filter_list_modal/delete_filter_list_modal.js

```diff
@@ -103,6 +103,7 @@ export class DeleteFilterListModal extends Component {
             selectedFilterLists.length === 0 ||
             canDeleteFilter === false
           }
+          data-test-subj="mlFilterListsButtonDelete"
```

Comment:
Nit - the convention used elsewhere is to end the data-test-subj with `Button` e.g. `mlCalendarEventDeleteButton`. Is there a reason for the different structure here?

Reply:
Fixed in 9fd1368 and afc8dab